docs/user/benchmarks.md

```diff
@@ -5,7 +5,7 @@ hivemind.
 
 ### Server throughput
 
-You can use [this benchmark](https://github.com/learning-at-home/hivemind/blob/master/tests/benchmark_throughput.py) to
+You can use [this benchmark](https://github.com/learning-at-home/hivemind/blob/master/benchmarks/benchmark_throughput.py) to
```

Comment:
Thanks, this move was long overdue :)

p.s. I've added a task to re-run the benchmark numbers to https://github.com/learning-at-home/hivemind/issues/232